dnf5/commands/environment/environment_install.hpp

```diff
@@ -0,0 +1,30 @@
+// Copyright Contributors to the DNF5 project
+// SPDX-License-Identifier: GPL-2.0-or-later
```

Comment:
I think the team decided that new files in dnf5 will use LGPL-2.1-or-later license.
Or did you base this new file on group/group_install.hpp?